src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -24,6 +24,8 @@ class DateTimeNormalizer implements NormalizerInterface, DenormalizerInterface,
 {
     const FORMAT_KEY = 'datetime_format';
     const TIMEZONE_KEY = 'datetime_timezone';
+    // Ignore timezone information supplied in time string when default timezone is set to ensure consistent timezone. This flag ensures backwards compatibility.
```

Comment:
This comment can probably be removed since it repeats the information stated elsewhere.